actions/lib/action.py

```diff
@@ -10,4 +19,10 @@ def __init__(self, config):
 
     def do_action(self, cls, action, **kwargs):
         obj = getattr(digitalocean, cls)(token=self._token)
-        return getattr(obj, action)(**kwargs)
+        digitalocean_results = getattr(obj, action)(**kwargs)
+        # The results received from the digitalocean module are
+        # custom python objects that StackStorm doesn't recognize.
+        # The following code converts them into dicts & lists
+        json_str = json.dumps(digitalocean_results, default=digitalocean_obj_to_dict)
```

Comment:
I think this would also throw if an object contains an attribute which is `datetime` class instance or instance of another class which is not JSON serializable, right?